docs/src/main/sphinx/release/release-453.md

```diff
@@ -0,0 +1,137 @@
+* Fix planning failure for queries with a filter on an aggregation. ({issue}`22716`)
+* Fix planning failure for queries involving multiple aggregations and `CASE`
+  expressions. ({issue}`22806`)
+* Fix optimizer timeout for certain queries involving aggregations and `CASE`
+  expressions. ({issue}`22813`)
+
+## Security
+
+* Extend `DROP ROLE` to allow `IF EXISTS`. ({issue}`21985`)
+* Improve performance for column-masking with [](/security/opa-access-control). ({issue}`21359`)
+
+## JDBC driver
+
+* Add support to use a certificate from the operating system keystore. ({issue}`22341`)
+* Add support for setting the default [SQL PATH](/sql/set-path). ({issue}`22703`)
+* Allow Trino host URI specification without port for the default ports 80 for
+  HTTP and 443 for HTTPS. ({issue}`22724`)
+
+## CLI
+
+* Add support to use a certificate from the operating system keystore. ({issue}`22341`)
+* Add support for setting the default [SQL PATH](/sql/set-path). ({issue}`22703`)
+* Allow Trino host URI specification without port for the default ports 80 for
+  HTTP and 443 for HTTPS. ({issue}`22724`)
+
+## BigQuery connector
+
+* Improve performance for querying table definitions. ({issue}`22770`)
+
+## Cassandra connector
+
+* Add support for the `execute` procedure. ({issue}`22556`)
+
+## ClickHouse connector
+
+* Add support for the `execute` procedure. ({issue}`22556`)
+
+## Delta Lake connector
+
+* Add support for reading partitioned tables which the partition column type is
+  changed by [type widening](https://docs.delta.io/latest/delta-type-widening.html).
+  ({issue}`22433`)
+* Allow using a Kubernetes service account on Amazon EKS for authentication to
+  Glue with `hive.metastore.glue.use-web-identity-token-credentials-provider`
+  set to `true`. ({issue}`15267`)
+* Fix support for SQL routine storage in Glue when
+  `hive.metastore.glue.catalogid` is set. ({issue}`22813`)
+
+## Druid connector
+
+* Add support for the `execute` procedure. ({issue}`22556`)
+
+## Exasol connector
+
+* Add support for the `execute` procedure. ({issue}`22556`)
+
+## Hive connector
+
+* Allow using a Kubernetes service account on Amazon EKS for authentication to
+  Glue with `hive.metastore.glue.use-web-identity-token-credentials-provider`
+  set to `true`. ({issue}`15267`)
+* Fix support for SQL routine storage in Glue when
+  `hive.metastore.glue.catalogid` is set. ({issue}`22813`)
+
+## Hudi connector
+
+* Allow using a Kubernetes service account on Amazon EKS for authentication to
+  Glue with `hive.metastore.glue.use-web-identity-token-credentials-provider`
+  set to `true`. ({issue}`15267`)
+* Fix support for SQL routine storage in Glue when
+  `hive.metastore.glue.catalogid` is set. ({issue}`22813`)
+
+## Iceberg connector
+
+* Add support for views with the JDBC catalog. ({issue}`22576`)
+* Upgrade `jdbc.schema-version` for the JDBC catalog to `V1`. ({issue}`22576`)
+* Add support for setting partition required schemas with the
```

Comment:
"Add support for specifying on which schemas to enforce the presence of a partition filter in query. This can be configured via the xxx option"